collections/_posts/2022-10-17-DiffusionModel.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: review
+title: "Deep Unsupervised Learning using Nonequilibrium Thermodynamics"
+author: "Sophie Carneiro Esteves"
+cite:
+    authors: "Jascha Sohl-Dickstein, Eric A. Weiss, Niru Maheswaranathan and Surya Ganguli"
+    title:   "Deep Unsupervised Learning using Nonequilibrium Thermodynamics"
+    venue:   "International Conference on Machine Learning"
+pdf: "http://proceedings.mlr.press/v37/sohl-dickstein15.pdf"
+---
+
+# Notes
+- Code is available on GitHub: [https://github.com/Sohl-Dickstein/
+Diffusion-Probabilistic-Models](https://github.com/Sohl-Dickstein/
+Diffusion-Probabilistic-Models)
+
+# Introduction
+The authors propose a new probalistic model that aims to link a known distribution $$ \pi(.) $$ to another one $$ q(.) $$ based on diffusion. To achieve it, they iteratively added small amount of noise on the target distribution until it is converted into the known distribution. This process can be seen has a Markov chain that can be learn reversly. 
+
+
+# Highlights
+- Flexible and tractable method
+- Possibility to multiply distributions (ex : inpainting)
+
+
+# Diffusion probalistic model
+
+> "A Markov chain or Markov process is a stochastic model describing a sequence of possible events in which the probability of each event depends only on the state attained in the previous event." [^1]
+
+A diffusion probalistic model consists in building a markov chain which allows to switch from a simple known distribution $$ \pi(.) $$ ( ex :  a gaussian distribution ) to a target distribution  $$ q(.) $$ (ex : a kind of image distribution).
+To obtain that chain they start from the target distribution and add small perturbations a thousand of times (eq. time steps) until we obtain the known distribution. This is the **forward trajectory**. 
```

Comment:
- Change from _we_ to _they_
- You cannot say _a thousand of times_. Do you mean _a thousand times_ exactly, or rather _thousands of times_. I would argue for _thousands of times_, since it's less specific and therefore probably more accurate to what is done in practice :wink: 
```suggestion
To obtain that chain they start from the target distribution and add small perturbations thousands of times (eq. time steps) until they obtain the known distribution. This is the **forward trajectory**. 
```